Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN npm install
 COPY . .
 
 RUN ./node_modules/.bin/gulp build --production && \
-    ./manage.py collectstatic --noinput
+    DJANGO_SETTINGS_MODULE=fala.settings.circle ./manage.py collectstatic --noinput
```

Comment:
Using `DJANGO_SETTINGS_MODULE=fala.settings.circle` in the `Dockerfile` we use to build a production image looks a bit weird.

I would assume we need this because the default behaviour is to "fail if things are missing"?

What do you think about flipping the relationship and make only the production environment fail for missing variables?

Reply:
> would assume we need this because the default behaviour is to "fail if things are missing"
> ?
Yes

Makes sense. I added a fala.settings.production, this is set in the DJANGO_SETTINGS_MODULE env var in the deployment.yml